src/ui/battle-message-ui-handler.ts

```diff
@@ -3,11 +3,11 @@ import { addBBCodeTextObject, addTextObject, getTextColor, TextStyle } from "./t
 import { Mode } from "./ui";
 import * as Utils from "../utils";
 import MessageUiHandler from "./message-ui-handler";
-import { getStatName, Stat } from "../data/pokemon-stat";
 import { addWindow } from "./ui-theme";
 import BBCodeText from "phaser3-rex-plugins/plugins/bbcodetext";
 import {Button} from "#enums/buttons";
 import i18next from "i18next";
+import { Stat, getStatKey, PERMANENT_STATS } from "#app/enums/stat.js";
```

Comment:
```suggestion
import { Stat, PERMANENT_STATS, getStatKey } from "#app/enums/stat";
```
NIT: Label types and standardize order